docs/release_notes/2024/sept2024.md

```diff
@@ -9,6 +9,27 @@ tags:
   - september
 ---
 
+## 3.3.11.\* (September 20, 2024)
+
+- Prophecy Python libs version: 1.9.14
+- Prophecy Scala libs version: 8.1.4
+
+### Features {#Features3311}
+
+#### Spark Copilot Enhancements {#Spark3311}
+
+- **Config syntax**: We've unified the config syntax across all project types by supporting Jinja config syntax. This means that you can use `{{foo}}` for variable substitution.
```

Comment:
For[ visual language ](https://docs.prophecy.io/Spark/configuration/#sql-visual-language) we'll need to update the docs.

Reply:
Currently working on. Thank you for the comment and link.

Reply:
Added more details.